src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import { Tooltip } from './Tooltip'
+
+describe('Tooltip component', () => {
```

Comment:
Could we add some tests that test:
- the Tooltip is hidden by default
- handling showing/hiding the Tooltip on the relevant JS events (mouseenter, focus, mouseleave, blur, keydown)

Reply:
More tests were added in 112fcd83166bac4c0d82dc579503d5823a3c10cd. Thanks!

Please still also add these tests (for keyboard navigation, `hover` doesn't cover these). 
- hides on blur
- shows on focus